src/system.ts

```diff
@@ -174,12 +175,39 @@ export class System {
   /**
    * Returns a file path.
    *
+   * @param options Options object.
+   * @param options.ext Number of generated extensions. Defaults to `1`.
+   *
+   * @throws When `options.ext < 0`
+   *
    * @example
-   * faker.system.filePath() // '/usr/local/src/money.dotx'
+   * faker.system.filePath()           // '/usr/local/src/money.dotx'
+   * faker.system.filePath({ ext: 0 }) // '/usr/local/src/money'
+   * faker.system.filePath({ ext: 2 }) // '/usr/local/src/money.dotx.zip'
    */
-  // TODO @prisis 2022-01-25: add a parameter to have the possibility to have one or two ext on file.
-  filePath(): string {
-    return `${this.directoryPath()}/${this.fileName()}`;
+  filePath(options?: { ext?: number }): string {
+    const ext = options.ext || 1;
+    const path = `${this.directoryPath()}/${this.fileName()}`;
+
+    switch (true) {
+      case ext < 0:
+        throw new FakerError('Options.ext shall not have negative value');
```

Comment:
```suggestion
        throw new FakerError('Options.ext should not be negative');
```
I think this phrasing would be more natural